renpy/main.py

```diff
@@ -550,7 +550,9 @@ def main():
 
         renpy.game.exception_info = 'While executing init code:'
 
-        for _prio, node in game.script.initcode:
+        for id_, (_prio, node) in enumerate(game.script.initcode):
```

Comment:
Maybe I'm stupid, but isn't this code adding things to a list we're already iterating ?
This seems to work in the console, but I'm not sure it's documented and/or reliable. In particular, sets and dicts react badly to that, and references I could find seem to indicate it as a very bad thing to do.
If it is the case, we could solve the problem by iterating with while, and either incrementing an index variable, or doing .pop(0) if we don't care about destroying the list.

Reply:
Indeed, we're changing the list we're iterating through, but in a smart way! You know the common pattern where you iterate a list in the reverse order to be able to `pop` elements from it? Similar approach is used here, we're adding elements to the list, but we're adding them *ahead* of the current index, which is totally safe.